src/index.ts

```diff
@@ -578,7 +578,10 @@ const _P2P = or(
  */
 export const P2P = fmt(_P2P)
 
-const _Circuit = and(_P2P, literal('p2p-circuit'), peerId())
+const _Circuit = or(
+  and(_P2P, literal('p2p-circuit'), peerId()),
+  and(_P2P, literal('p2p-circuit'), literal('webrtc'), optional(peerId())) // webrtc relay
```

Comment:
This module has two types of matching - `.matches` and `.exactMatch`

An address that `.matches` can have additional path components that might make it `.matches` other address types.  `.exactMatch` means the address is unequivocally of that type.

E.g.:

```js
const address = multiaddr('/dns/example.com/tcp/80/ws')

TCP.matches(address) // true, starts with a TCP address
TCP.exactMatch(address) // false is a WebSocket address
```

By adding the `literal('webrtc')` component, it will make `Circuit.exactMatch` return true for WebRTC addresses which would be a bug.

What you could do is make the `peerId()` optional:

```js
const _Circuit = and(_P2P, literal('p2p-circuit'), optional(peerId()))
```

...but then you wouldn't actually have a valid circuit relay address since you need the final peer id to tell the relay which peer you're trying to reach.